### ISSUE TITLE
OC 0.9.5 coreaudiod preventing sleep and change in Focus darkwake macos

### ISSUE THREAD

**Issue by tucomel** [seven-of-eleven/designare-z390-opencore-efi#78]
**Describe the bug**
Im using ventura 13.6 with your EFI but with smbios 19.1
Im facing 2 issues with sleep/wake 
First one is with coreaudiod PreventUserIdleSystemSleep = 1 : 
 * qemu-system-x86-64 (android emulator) is preventing macos from sleep (I had to disable the audio output and input in config.ini file to achieve  user idle sleep)
 * backtrack (audio record app) which uses the same mechanism of coreaudiod is preventing macos from sleep (even while im nothing use microphone or not playing any audio from system)

I've been using these 2 apps for many years on other versions of macos (monterey and ventura13.1) and I've never had this problem, it only appeared after updating to version 13.6 and also updating the kexts (from your EFI), my bet would be some change of standards in AppleALC or the need to apply a patch to ACPI tables...

The second problem is about Focus mode :
I have a sleep focus mode set on my iPhone that turns ON every day at midnight and turns OFF at 9:30 am, and this exacly time the macos wakes up to synchronize this change of focus mode and then it goes back to sleep.. the log of this wake is below 

```
2023-10-17 03:03:31 -0300 Sleep               	Entering Sleep state due to 'Idle Sleep': Using AC (Charge:0%) 23284 secs
2023-10-17 09:31:35 -0300 DarkWake            	DarkWake from Normal Sleep [CDN] : due to RTC/Maintenance Using AC (Charge:0%) 45 secs <-- wake from focus change on my iphone 
2023-10-17 09:32:20 -0300 Sleep               	Entering Sleep state due to 'Maintenance Sleep': Using AC (Charge:0%) 11294 secs
2023-10-17 12:40:34 -0300 Wake                	Wake from Normal Sleep [CDNVA] : due to XDCI/UserActivity Assertion Using AC (Charge:0%) <-- I wake using keyboard
```
I would like to know if there is a way for MacOS not to wake up to synchronize focus mode changes came from iphone o apple watch in case they exists.

**OpenCore Logs**
Log was captured using opencore 0.9.5 debug mode
LOG FILE IS TOO LONG TO POST HERE, it is attached  bellow
[opencore-2023-10-17-230546.txt](https://github.com/seven-of-eleven/designare-z390-opencore-efi/files/12968838/opencore-2023-10-17-230546.txt)

**Expected behavior**
If there is no playing audio from microphone or system audio , the coreaudiod controlller should receive the UserIdle command and accept going into sleep mode

**Screenshots**
print from sleep aid app, for testing I set the computer to sleep after 1min, and after 1min monitors got off but the system is still on, for more 3:20 seconds, handled by the coreaudiod process..
![image](https://github.com/seven-of-eleven/designare-z390-opencore-efi/assets/8607299/56e7e045-cc9a-4ebc-8fdf-10b1de2cd62e)

**Hardware Specifications (please complete the following information):**
 - CPU: Intel Core i9 9900KS
 - GPU: rx5700xt
 - 32gb ram trident 3200mhz
 - Motherboard: Designaire Z390
 - Storage: 
![image](https://github.com/seven-of-eleven/designare-z390-opencore-efi/assets/8607299/1370e379-ee9f-40db-b612-b4b02f58b060)

**Additional context**

**Comment by seven-of-eleven** [comment]
@tucomel thanks for information. I'll do some digging to see if I can resolve the issues and post back here as I go.

**Comment by tucomel** [comment]
just to you know, I disabled this setting and the darkwakes stopped wakes the computer when the focus got changed remotelly 

![image](https://github.com/seven-of-eleven/designare-z390-opencore-efi/assets/8607299/fac8caae-3b47-473f-af11-5148cddfb011)

**Comment by seven-of-eleven** [comment]
@tucomel Thanks for the update. Apparently this also happens on Apple hardware. I found a number of people complaining about it. What you've suggested above was one of the solutions mentioned (I've also disabled it). Others options involve essentially disabling anything todo with syncing between devices,  wake on lan... I also found the following commands to help determine what's keeping your device awake:

`pmset -g` - shows the current power settings and what's preventing sleep
`pmset -g log | grep -e " Sleep " -e "Wake "` - shows log details
`pmset -g assertions` - shows more details than above

I found out Arc browser was keeping my Mac awake (fixed after restarting the app).

Learned something new again. Thanks for bringing this up. All the best.
